### PR TITLE
Environmental solutions

### PR DESCRIPTION
**Pull request by baraabourghli**

### CODE REVIEW

File: app/templates/environmental-solutions.hbs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<div class="header">
+    <h1>Environmental Solutions</h1>
+</div>
+<div class="content">
+    <form class="pure-form search" {{action "search" on="submit"}}>
+        <fieldset>
+            <div class="pure-g">
+                <div class="pure-u-1">
+                    <label>Keyword</label>
+                  {{input class="pure-input-1" value=qField}}
```
Comment (baraabourghli) [inline app/templates/environmental-solutions.hbs]: About `value=qField`, eccn implementation as a guide, I can see the definition is done here https://github.com/GovWizely/explorer/blob/master/app/controllers/eccns.js#L7.
I suppose in my case it should be exactly the same correct?

**Comment by baraabourghli** [comment]
@calumhalcrow I think this is ready for review.

File: app/templates/environmental-solutions.hbs (hunk 1 above)
Comment (baraabourghli) [inline app/templates/environmental-solutions.hbs]: I thought this was a special name related to Ember, i think i know now what is going on.

**Comment by calumhalcrow** [comment]
Looks good! I'm pleasantly surprised that you got this all working without any problems and in such a short timeframe :smiley:

**Pull request merged**